src/app/core/layouts/header/header.component.html

```diff
@@ -14,5 +14,6 @@
       </ng-template>
     </p-menubar>
     <app-country-dropdown></app-country-dropdown>
+    <p-inputSwitch [(ngModel)]="checked" (click)="onThemeChange(checked ? 'dark' : 'light')"></p-inputSwitch>
```

Comment:
Do you need this ternary operator on the parameters of the method onThemeChange ?

Can't you, in the onThemeChange method body, just do a : 

```
this.themeService.setTheme(checked ? 'dark' : 'light');
```